### PR TITLE
Remove ZopeTestCase dependency.

### PR DESCRIPTION
**Pull request by tisto**

### CODE REVIEW

**Comment by tisto** [comment]
http://jenkins.plone.org/job/pull-request-5.0/5/

File: plone/app/customerize/tests/testCustomizeView.txt [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
 
-  >>> browser = self.site_administrator_browser
+First we have to set up some things and login.
+
+  >>> app = layer['app']
+  >>> from plone.testing.z2 import Browser
+  >>> browser = Browser(app)
   >>> browser.handleErrors = True
+  >>> browser.raiseHttpErrors = True
+  >>> browser.addHeader('Authorization', 'Basic admin:secret')
```
Comment (ebrehault) [inline plone/app/customerize/tests/testCustomizeView.txt]: Ok so that's the thing I missed apparently.
Thank you for fix that.

Comment (tisto) [inline plone/app/customerize/tests/testCustomizeView.txt]: Unfortunately it seems this is not enough. five.customerize is not installed properly with your new test fixture. This is why this solves the test isolation problems.

**Comment by tomgross** [comment]
I guess this is superseeded by https://github.com/plone/plone.app.customerize/pull/7
@tisto ?

**Comment by jensens** [comment]
I think so too and close this one, please reopen if i'am wrong

**Pull request closed without merging**